build.gradle

```diff
@@ -26,6 +26,11 @@ allprojects {
     resolutionStrategy {
       force 'org.codehaus.groovy:groovy-all:2.4.5'
       force 'org.spockframework:spock-core:1.0-groovy-2.4'
+      eachDependency {
+        if (it.requested.group == 'org.springframework') {
+          it.useVersion spinnaker.version('spring')
```

Comment:
spring-session-data-redis is a patch release or two different than the version spring-boot brings in, so this will keep them in sync for now